tests/data/DISABLED

```diff
@@ -70,9 +70,19 @@
 266
 579
 587
+# most of the IPFS tests, hyper doesn't like them
```

Comment:
Why doesn't hyper like the IPFS? This seems like something we should fix, not hide.

Reply:
It's the CRLF vs LF line endings. I just can't get those to work.
If you have more luck, please do. Last time i spend days trying to make hyper like it, nothing worked.

Reply:
Hyper is fixed. I just needed to add `nocheck="yes"` to all tests that contain server data. I somehow completely missed that before. No IPFS tests are disabled for hyper now. Yay :)